README.md

```diff
@@ -20,11 +21,12 @@ Generic File Submission API (to be used by the Family Law Act Application at fir
 
 ## Apps
 
-| Name             | description                                  | doc                                             |
-| ---------------- | -------------------------------------------- | ----------------------------------------------- |
-| backend          | all server side services                     | [README](src/backend/README.md) |
-| jag-efiling-api  | the main api for interating with the service | [README](src/backend/jag-efiling-api/README.md) |
-| efiling-frontend | the frontend for uploading documents         | [README](src/efiling-frontend/README.md)        |
+| Name                    | description                                  | doc                                             |
+| ----------------        | -------------------------------------------- | ----------------------------------------------- |
+| backend                 | all server side services                     | [README](src/backend/README.md) |
+| jag-efiling-api         | the main api for interating with the service | [README](src/backend/jag-efiling-api/README.md) |
+| file-submission-worker  | the main api for interating with the service | [README](src/backend/file-submission-worker/README.md) |
```

Comment:
### Question
The description for this is same as for `jag-efiling-api`, doesn't seem like this should be the case. Can we provide a better description for this worker here?